JeMPI_Apps/JeMPI_LibAPI/src/main/java/org/jembi/jempi/libapi/BackEnd.java

```diff
@@ -537,6 +553,12 @@ public record GetGidsAllRequest(ActorRef<GetGidsAllResponse> replyTo) implements
    public record GetGidsAllResponse(List<String> records) implements EventResponse {
    }
 
+   public record GetConfigurationRequest(ActorRef<GetConfigurationResponse> replyTo) implements Event {
+   }
+
+   public record GetConfigurationResponse(String configuration) implements EventResponse {
+   }
+
```

Comment:
@harmochiky this response type and its properties need to be more explicit than bundling it all into a string. With the current solution the file dictates the format of the response. What happens when the file format is incorrectly saved?